app/api/v2/handlers/schedule_api.py

```diff
@@ -0,0 +1,131 @@
+import aiohttp_apispec
+from aiohttp import web
+
+from app.api.v2.handlers.base_object_api import BaseObjectApi
+from app.api.v2.managers.schedule_api_manager import ScheduleApiManager
+from app.api.v2.responses import JsonHttpForbidden
+from app.api.v2.schemas.base_schemas import BaseGetAllQuerySchema, BaseGetOneQuerySchema
+from app.objects.c_schedule import Schedule, ScheduleSchema
+
+
+class ScheduleApi(BaseObjectApi):
+    def __init__(self, services):
+        super().__init__(description='schedule', obj_class=Schedule, schema=ScheduleSchema, ram_key='schedules',
+                         id_property='name', auth_svc=services['auth_svc'])
+        self._api_manager = ScheduleApiManager(services)
+        self._rest_svc = services['rest_svc']
+
+    def add_routes(self, app: web.Application):
+        router = app.router
+        router.add_get('/schedules', self.get_schedules)
+        router.add_get('/schedules/{name}', self.get_schedule_by_name)
```

Comment:
to be consistent with other API objects, we should replace name with `id`. The inner operation should already have a name anyway, and this will help avoid issues like folks putting spaces in the name, etc.

Reply:
A schedule will only handle one operation at a time. The operation would be created at the same time as the schedule, so you wouldn't have to worry about a single operation being linked to more than one schedule